web/libs/editor/tests/e2e/plugins/errorsCollector.js

```diff
@@ -50,6 +50,22 @@ const IGNORE_ACTION = "ignore";
 const DISPLAY_ACTION = "display";
 const INTERRUPT_ACTION = "interrupt";
 
+/**
+ * Safely get the jsonValue of a JSHandle, handling "Target closed" and other errors.
+ * @param {any} arg - The JSHandle to get the value from
+ * @returns {Promise<any>} - The value, or undefined if target closed, or a fallback string on other errors
+ */
+async function safeJsonValue(arg) {
+  try {
+    return await arg.jsonValue();
+  } catch (error) {
+    if (error.message && error.message.includes("Target closed")) {
+      return undefined;
+    }
+    return "[Error reading console argument]";
```

Comment:
should we add more details to this? I don't like messages like "something went wrong" when we should know exactly what went wrong